sunpy/net/jsoc/jsoc.py

```diff
@@ -64,7 +74,11 @@ def __len__(self):
             return len(self.table)
 
     def __getitem__(self, item):
-        return type(self)(self.table[item])
+        type_self = type(self)
```

Comment:
I think I would rather write this as:

```suggestion
        resp = type(self)(self.table[item])
```

Reply:
Great! thanks!